Philips.CodeAnalysis.MsTestAnalyzers/TestMethodNameAnalyzer.cs

```diff
@@ -37,15 +37,22 @@ private static void Analyze(SyntaxNodeAnalysisContext context)
 			foreach (AttributeSyntax attribute in attributesNode.Attributes)
```

Comment:
Please replace this with Linq using Any(attr => attr.Name.ToString() == @"TestMethod"), as this will bail out early and is easier to read.

Reply:
This is beyond the scope of this PR, which is already flooded with changes to accommodate IntelliSense. Will create a separate Issue for this.